assets/js/llms-metabox-voucher.js

```diff
@@ -143,10 +143,17 @@
 	 */
 	function check_voucher_duplicate() {
 
+		var vouchers = get_codes_from_inputs();
+
+		if( ! vouchers.length ) {
+			jQuery( "#post" ).submit();
+			return;
+		}
+
 		var data = {
-			action: 'check_voucher_duplicate', 'postId' :
-			jQuery( '#post_ID' ).val(),
-			'codes' :  get_codes_from_inputs(),
+			action: 'check_voucher_duplicate',
+			'postId': jQuery( '#post_ID' ).val(),
+			'codes':  get_codes_from_inputs(),
```

Comment:
```suggestion
			'codes':  vouchers,
```

And then please update the function docblock's changelog :D

Reply:
Code updated, thanks 🥲